.travis.yml

```diff
@@ -30,25 +32,19 @@ addons:
 
 cache:
   directories:
-    - $HOME/.cache/support
-    - $HOME/.cache/iocs
     - $HOME/.cache/pip
     - $HOME/.ccache  # https://github.com/travis-ci/travis-ci/issues/5853
 
 before_install:
   - export PATH=/usr/lib/ccache:$PATH
-  - export EPICS_CA_ADDR_LIST=127.255.255.255
-  - export EPICS_CA_AUTO_ADDR_LIST=NO
-  - export EPICS_CA_MAX_ARRAY_BYTES=10000000
-  - ip addr
+
   - git fetch --unshallow
-  - source .ci/epics-config.sh
-  - bash .ci/install-epics-base.sh
-  - bash .ci/install-epics-modules.sh
-  - bash .ci/install-epics-iocs.sh
-  - bash .ci/run-epics-iocs.sh
+  - export CI_SCRIPTS=${TRAVIS_BUILD_DIR}/.ci/ci-scripts
+  - source ${CI_SCRIPTS}/epics-config.sh
+  - bash .ci/install-from-release.sh "https://github.com/klauer/epics-on-travis/releases/download/${PACKAGED_DEPS}"
```

Comment:
Is this supposed to be `NSLS-II` now, not `klauer`?

Reply:
I didn't get the releases on NSLS-II just yet - still need to setup the API key and re-trigger a build on travis.